docs/src/manual/boundary_conditions.md

```diff
@@ -10,20 +10,85 @@ the most common ones, Dirichlet and Neumann boundary conditions, and how to do i
 
 ## Dirichlet Boundary Conditions
 
-At a Dirichlet boundary the solution is prescribed to a given value. For the discrete
-FE-solution this means that there are some degrees of freedom that are fixed. To be able
-to tell which degrees of freedom we should constrain we need the `DofHandler`.
+At a Dirichlet boundary the unknown field is prescribed to a given value. For the discrete
+FE-solution this means that there are some degrees of freedom that are fixed. To handle
+Dirichlet boundary conditions in Ferrite we use the [`ConstraintHandler`](@ref). A
+constraint handler is created from a DoF handler:
 
 ```julia
 ch = ConstraintHandler(dh)
 ```
 
-TBW
+We can now create Dirichlet constraints and add them to the constraint handler. To create a
+Dirichlet constraint we need to specify a field name, a part of the boundary, and a function
+for computing the prescribed value. Example:
+
+```julia
+dbc1 = Dirichlet(
+    :u,                       # Name of the field
+    getfaceset(grid, "left"), # Part of the boundary
+    (x, t) -> 1.0 * t,        # Function mapping coordinate and time to a prescribed value
+)
+```
+
+The field name is given as a symbol, just like when the field was added to the dof handler,
+the part of the boundary where this constraint is active is given as a face set, and the
```

Comment:
```suggestion
the part of the boundary where this constraint is active is given as a `<:BoundaryIndex` set, and the
```
I think we can also do node and edge sets, can't we?

Reply:
Yea I suppose, but feels slightly more advanced/unusual. Perhaps it could be a subsection about it or something.